Maquetado/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
```

Comment:
As i mentioned above we use two space indentation, and here do not leave this empty spaces. Also, leave a blank line at the end of the file.